src/credentials/BUILD.gn

```diff
@@ -157,6 +158,8 @@ static_library("default_attestation_verifier") {
     "attestation_verifier/DefaultDeviceAttestationVerifier.cpp",
     "attestation_verifier/DefaultDeviceAttestationVerifier.h",
     "attestation_verifier/DeviceAttestationDelegate.h",
+    "attestation_verifier/TestDACRevocationDelegateImpl.cpp",
+    "attestation_verifier/TestDACRevocationDelegateImpl.h",
```

Comment:
Remove these files from here. and the `jsoncpp_root` dependency in default_attestation_verifier

Reply:
Thanks for pointing it out, removed. Thats why tests were passing!